### ISSUE TITLE
Add an option to show window from the taskbar

### ISSUE THREAD

**Issue by whizzzkid** [ipfs/ipfs-desktop#2300]
**Is your feature request related to a problem? Please describe.**
When IPFS desktop starts, the main window disappears and the icon exists on the taskbar. Since the window is hidden, there is no way to access the UI unless an option like `Preferences` or `Advanced` is selected.

**Describe the solution you'd like**
Have a simple `Show Window` button to display the UI. At least on mac, the use on other OSes is not clear.

<img width="211" alt="show window" src="https://user-images.githubusercontent.com/1895906/195792803-518f5e07-1291-47af-a72d-449b3fa9b1c8.png">

**Comment by SgtPooki** [comment]
1. IPFS desktop being opened, when one is already running, should open/bring-to-front the existing desktop that is running
2. IPFS desktop actions for the task bar should make it much more obvious how to bring desktop UI to the front

@juliaxbow do you have any thoughts on these?

**Comment by juliaxbow** [comment]
I agree with the solution described by @whizzzkid to include a "Show Window" button in the taskbar. 
- This button ("Show Window") should also be available in the context/dock menu accessed from the icon in the macOS dock.
- If we include a "Show Window" button, we should also include a "Hide" in both the taskbar and the macOS doc. This is standard across context menus
- I'd recommend including the "Show Window" and "Hide" buttons above "Quit" 

Side note/question, is there any reason that clicking on the icon in the taskbar doesn't open Desktop App UI? I think both clicking on the IPFS Desktop Icon _and_ selecting "Show Window" from the taskbar/dock menu should allow the user to see the desktop UI. The current behavior as described in this issue is unexpected and thus not a great user experience.

**Comment by SgtPooki** [comment]
@juliaxbow I don't know if there were any existing arguments against opening the app when the icon is clicked, but since we've already established a pattern with IPFS Desktop where we don't, I don't think we should add it now.

I'd prefer to keep the action explicit, as I believe the "tray icon only" operation mode is how more than a few users use it 
  * We need to be able to argue/support this with data from countly.
